pkg/injector/bootstrap.go

```diff
@@ -53,12 +50,12 @@ func (wh *mutatingWebhook) createEnvoyBootstrapConfig(name, namespace, osmNamesp
 		CipherSuites:          wh.configurator.GetMeshConfig().Spec.Sidecar.CipherSuites,
 		ECDHCurves:            wh.configurator.GetMeshConfig().Spec.Sidecar.ECDHCurves,
 	}
-	config, err := bootstrap.GenerateEnvoyConfig(configMeta, wh.configurator)
+	bootstrapConfig, err := builder.Build()
```

Comment:
@steeling Is it fair to say that `builder` in this context is specif to Envoy Bootstrap config?
It is not a generic builder - it always only builds config for Envoy?
If that's the case - I'd love to see `builder` renamed to something that is a bit higher specificity. It could be `envoyBootstrapBuilder` or `bootstrapBuilder`.  I can imagine having multiple builders in the same file/context and this would disambiguate.

Reply:
it's a few lines away from the instantiation,`builder := bootstrap.Builder`.

Did you see my prior comment above? Copying here:

> [I'll point out that go explicitly prefers short variable names when it is used in a limited scope](https://github.com/golang/go/wiki/CodeReviewComments#variable-names)
> 
> In this case, the method config, and builder both appear within a few lines of the function name createEnvoyBootstrapConfig, and the instantion of the builder: builder := bootstrap.Builder{...
> 
> I don't think it's too important for a single line to have that much context, but too quickly skim a handful of lines and understand what is happening.
> 
> Anyways, just wanted to add that as a counter point, I've changed to bootstrapConfig which should be a happy medium